server/server.go

```diff
@@ -106,15 +103,23 @@ func (s *Server) Teardown() {
 // in order to provide a nicer API when setting up the server.
 func (s *Server) EnableTracing() {
 	grpc.EnableTracing = true
-	logger.Notice("Enabled gRPC tracing")
+	s.logger.Notice("Enabled gRPC tracing")
+}
+
+func (s *Server) SetLogLevel(level string) error {
```

Comment:
I think we don't need this wrapper here, we are supposed to fully configure the logger prior even assigning it to the `Server`. Please remove.